manifests/init.pp

```diff
@@ -439,10 +439,11 @@
 #                                           Defaults to 30000, using the Jetty default of 30s
 #
 # $server_puppetserver_metrics::            Enable puppetserver http-client metrics
-#                                           Defaults to false because that's the Puppet Inc. default behaviour.
+#                                           Defaults to true because that's the Puppet Inc. default behaviour (since Puppetserver 5.0.0).
```

Comment:
I wonder if we even need to mention the default at all, but perhaps shorten it?
```suggestion
#                                           Defaults to true, matching defaults in Puppetserver 5+.
```

Reply:
Updated and rebased.